neurom/tests/test_stats.py

```diff
@@ -68,7 +70,11 @@ def test_fit_normal_regression():
     nt.assert_almost_equal(fit_.params[0], 10.019332055822, 12)
     nt.assert_almost_equal(fit_.params[1], 0.978726207747, 12)
     nt.assert_almost_equal(fit_.errs[0], 0.021479979161, 12)
-    nt.assert_almost_equal(fit_.errs[1], 0.745431659944, 12)
+
+    if sys.version_info[0] == 2:
```

Comment:
This failure depends on the version of scipy, but I think here I think you're checking the neurom version? alternative to this is to ensure that the same mode for kstest is used regardless of scipy version (e.g. #839 ).

Reply:
Since 3 days ago we dropped Python 2 so this is no longer an issue.

Reply:
I failed this test with python 3.7 after updating scipy, but this now passes CI so i dunno